File: game-server/src/main/java/org/forfun/mmorpg/game/admin/SimplyReply.java
package org.forfun.mmorpg.game.admin;

public class SimplyReply {

	private int status;

	private String msg;

	public static SimplyReply valueOfSucc(String msg) {
		SimplyReply reply = new SimplyReply();
		reply.msg = msg;
		return reply;
	}

	public static SimplyReply valueOfFailed(int code, String msg) {
		SimplyReply reply = new SimplyReply();
		reply.status = code;
		reply.msg = msg;
		return reply;
	}

	public int getStatus() {
		return status;
	}

	public void setStatus(int status) {
		this.status = status;
	}

	public String getMsg() {
		return msg;
	}

	public void setMsg(String msg) {
		this.msg = msg;
	}

}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/admin/SimplyReply.kt
package org.forfun.mmorpg.game.admin

class SimplyReply {

    var status: Int = 0
    var msg: String? = null

    companion object {
        // 简化构造，直接实例返回
        fun succ(msg: String?) = SimplyReply().apply { this.msg = msg }
        fun fail(code: Int, msg: String?) = SimplyReply().apply {
            status = code
            this.msg = msg
        }
    }
}
